seedlet/functions.php

```diff
@@ -396,7 +397,9 @@ function seedlet_skip_link_focus_fix() {
 /**
  * Custom colors class.
  */
-require get_template_directory() . '/classes/class-seedlet-custom-colors.php';
+if ( empty( get_theme_mod( 'colors_manager' ) ) ) {
```

Comment:
Can you add a code comment here? I can imagine someone not familiar with WordPress.com color annotations wondering what this is for.